src/styles/nodeAttributes.ts

```diff
@@ -79,17 +88,17 @@ export class NodeAttributes extends ItemAttributes {
           if (typeof rule.style.color === 'object') {
             const propValue = Tools.getIn(itemData, rule.style.color.input);
             if (Array.isArray(propValue)) {
-              c = ItemAttributes.autoColor(itemData.categories[i], rule.style.ignoreCase);
+              color = ItemAttributes.autoColor(itemData.categories[i], rule.style.color.ignoreCase);
```

Comment:
ignoreCase doesn't exist in type style but in color

Reply:
we never had an error because before it was always considered as undefined when passing it to getRandomUniqueColorPaletteIndex